pkg/object/annotations_test.go

```diff
@@ -0,0 +1,199 @@
+// Copyright 2021 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package object
+
+import (
+	"testing"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+var (
+	clusterScopedObj = ObjMetadata{Name: "cluster-obj",
+		GroupKind: schema.GroupKind{Group: "test-group", Kind: "test-kind"}}
+	namespacedObj = ObjMetadata{Namespace: "test-namespace", Name: "namespaced-obj",
+		GroupKind: schema.GroupKind{Group: "test-group", Kind: "test-kind"}}
+)
+
+var u1 = &unstructured.Unstructured{
```

Comment:
Nit: I think this might be more readable if we just put the KRM manifests as strings and then parse it into Unstructured. We do this in the applier tests.

Reply:
Sounds good. I think this is better handled by creating a testutil package with a bunch of standard KRM resources for all tests. That should be another PR.